qutip/visualization.py

```diff
@@ -372,7 +384,7 @@ def sphereplot(theta, phi, values, fig=None, ax=None, save=False):
     """
     if fig is None or ax is None:
         fig = plt.figure()
-        ax = Axes3D(fig)
+        ax = _axes3D(fig, azim=-35, elev=35)
```

Comment:
Is there a reason to change the arguments in this one?

Reply:
Sorry for missing all these intricacies! There was absolutely no reason for me to change that. Thanks again.

Reply:
That's no problem, it happens to everyone when we go back and forth between changes.  I think this is ready now, but we'll wait til the tests have officially passed.

Reply:
Yes, absolutely!